common/OpTestHMC.py

```diff
@@ -80,6 +97,41 @@ def check_kernel_signature(self):
             return "signed_kernel"
         return "unsigned_kernel"
 
+    def get_grub_file(self):
+        '''
+        Fetch the grub-file based on the distro
+        '''
+        if self.distro == 'rhel':
+            cmd = "rpm -ql grub2-ppc64le"
+            out = self.console.run_command(cmd)
+            for line in out:
+                if 'core.elf' in line:
+                    grub_filename = line
+            if not grub_filename:
+                self.fail("%s: Failed to get grub file" % (self.distro))
+        elif self.distro == 'suse' or self.distro == 'SLES':
+            cmd = "rpm -ql grub2-powerpc-ieee1275"
+            out = self.console.run_command(cmd)
+            for line in out:
+                if 'grub.elf' in line:
+                    grub_filename = line
+            if not grub_filename:
+                self.fail("%s: Failed to get grub file" % (self.distro))
+        else:
+            raise self.skipTest("Test currently supported on SLES and RHEL releases")
```

Comment:
I think statment is wrong here, when it is supported why do we skip it.

Reply:
Right, ideally I meant it to be - _"Test currently supported on SLES & RHEL realeases **only**"_
Will change it.